### ISSUE TITLE
Objects  disappear in 3D Repo with tasks that overlap 

### ISSUE THREAD

**Issue by Ramadyfm** [3drepo/3drepobouncer#618]
### Description
This bug is seen when assigning  objects to a resource that has two overlapping tasks created in Synchro. The objects are seen to disappear at different points of time in the sequence in 3D Repo when they should remain visible.

This bug is occurring in: Production
This requires a hotfix to production: TBD
Steps to replicate

- Created 3 objects in synchro

- Duplicated the objects 3 times 

- Assign all objects to resource (called Cylinder and Box) 

- Create 2 different overlapping tasks from the resource (Task 1 : "Cylinder & Box" (this task has no transformation) and Task 2 : "Cylinder & Box Transformation" (this task has a transformation))

- .sp file can be downloaded [here ](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/EWYiP7-QUsBOkzz7nENUf5MB3sOZOU9qUskf6rFpsDwk5g?e=mTA4Ac)and exported to 3D Repo.

- .spm file can be downloaded [here ](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/EVgWaWoOMIdEqcvJ3c--S_YBnMHvYzn-W3T4R-N0_2UP0g?e=Y47zmw) and uploaded directly to 3D Repo

Specific focus times:

.spm file before task 1 can be found [here](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/EeNDbY8D-nZGujdrwr_XJVcBilCsCiB64JMTFn1f4yv1rw?e=2wLFSP)
.spm file during task 1 and before overlap can be found [here](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/EaJPcFVhnd5AhPX74ho0L-YBfw1elztN8b2UWmrQCZor8Q?e=KlXb5R)
.spm file during overlap between task 1 and 2 can be found [here](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/ESrBGvQD2OdDkWJ0bKYe2fQBqS7tV1PzYAGNZREOY2QYHQ?e=z3ajIw)
.spm file during task 2 and after overlap can be found [here](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/EQg1ifA75DVCqNlhMELKpW0Bqv6mRZdpEke-Fdf41Q2KYA?e=QgF3SH)
.spm file after task 2 can be found [here](https://asitesol.sharepoint.com/:u:/s/3DRepoTeamFolder/EQdYTWGuBZVDqP3Zt_9ejSoBxDD36bfdiIK2sVySTQk-WQ?e=THzDSe)

### Current Behaviour

- Box and cylinders  from  task 1, disappear during the overlap of task 1 & 2 

- Box and cylinders from task 2  disappear after the overlap 
### Expected Behaviour 

- Box and cylinders from task 1 remain during overlap 

- Box and cylinders from task 1 to disappear and task 2 to remain  after overlap 

In Synchro:
During overlap
![image](https://user-images.githubusercontent.com/100203626/225586565-72666e39-7e16-43f3-a4d1-cd57d4fd5aa3.png)

After Overlap:

![image](https://user-images.githubusercontent.com/100203626/225586833-480cbafb-d531-4ab0-81ce-64afa4f63abc.png)

In 3D Repo:

During overlap

![image](https://user-images.githubusercontent.com/100203626/225587239-33e9b1bb-b2c7-4f13-9cc2-60f6d63d7236.png)

After overlap

![image](https://user-images.githubusercontent.com/100203626/225587389-1b616ad0-1527-4f51-88b4-fc1e1186866e.png)

**Comment by carmenfan** [comment]
2 things I can see happening, one I can fix and one I can't:

1. This is probably not a problem @Ramadyfm ran into originally, which is that since v4.4  we started converting the model into units as dictated by the model settings, but we didn't scale the transformation accordingly - this I can fix

2. The actual problem here is similar to #582, SPM export doesn't correctly export the animation timeline when there are overlapping tasks involving the same resource ID. This is something we can't fix as what you see is what we got from synchro: 
![image](https://github.com/3drepo/3drepobouncer/assets/11945337/f7d2191e-c692-4f7a-8e40-46b5f5308484).
